src/testdata/stellar-history.testnet.6714240.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
I recommend moving the testing of the new file format to its own commit *before* making any other code change, that way you can verify that the old version can indeed consume this new format.